Minor Changes in LocationList.py

RealLarsP opened the pull request:
I've edited a few small things like removing/adding empty lines and changed the area-name comments, so they match the rest (only empty lines between entire areas, and not sometimes also between e.g. Freestanding and Pots/Crates, and not using sometimes abbreviations for area names and sometimes using the full name). Also I sorted two checks in GTG MQ (because it was First, Third, Second before) and fixed two typos in a name of an unused pot in BotW MQ.

RealLarsP committed:
Minor Changes in LocationList.py

I've edited a few small things like removing/adding empty lines and changed the area-name comments, so they match the rest (only empty lines between entire areas, and not sometimes also between e.g. Freestanding and Pots/Crates, and not using sometimes abbreviations for area names and sometimes using the full name). Also I sorted two checks in GTG MQ (because it was First, Third, Second before) and fixed two typos in a name of an unused pot in BotW MQ.

cjohnson57 commented on LocationList.py:
```diff
@@ -267,7 +267,6 @@ def shop_address(shop_id: int, shelf_id: int) -> int:
     ("Market Bombchu Shop Item 8",                                   ("Shop",         0x32,  0x37, (shop_address(2, 7), None),      'Buy Bombchu (20)',                      ("Market", "Shops",))),
 
     ("ToT Light Arrows Cutscene",                                    ("Cutscene",     0xFF,  0x01, None,                            'Light Arrows',                          ("Temple of Time", "NPCs",))),
-
```
The spaces on the lines above (250, 259, 268) should also be removed to keep consistency

RealLarsP committed:
Update LocationList.py

RealLarsP committed:
Update LocationList.py

Just also swapped the two child-only chests in GC maze to make it left, center, right instead of left, right, center.
